.eslint-bin/pre-commit-hook.js

```diff
@@ -0,0 +1,39 @@
+const simpleGit = require('simple-git')(`${__dirname}/..`);
+let listFilesModified = [];
+
+function getFilesModified(callback) {
+  simpleGit.status((error, status) => {
+    if (error) {
+      console.error(error);
+      process.exit(-1);
+    }
+
+    listFilesModified = status.files
+      .map(file => file.path)
+      .filter(file => file.endsWith('.js'));
+
+    callback();
```

Comment:
[Code Convention] Could you return a list of file from this function for example in the callback and then use it instead of setting a global variable ? Or transform it to a promise ?